scripts/generate-changelog

```diff
@@ -183,9 +182,13 @@ def process_changelog(resource_type: str, changelog: dict, releases: set, projec
                 if resource_type == PULL_REQUESTS:
                     if '[bot]' in item['title']:
                         continue  # skip auto-gen PR
-                    no_project_prs.append(item)
-                continue
-            project = item['projectCards']['nodes'][0]['project']['name']
+                    # Obtain the version from the PR's target branch, e.g. branch-x.y --> x.y
+                    ver = item['baseRefName'].replace('branch-', '')
+                    project = f"{RELEASE} {ver}"
+                else:
+                    continue
+            else:
```

Comment:
nit: this should be safe to be removed, as it was trying make the script compatible for classic projects which was gone for a long time. its fine we could still leave it here

Reply:
Agree; removing compatibility with project cards simplifies the logic."